ext/financialacls/tests/phpunit/Civi/Financialacls/LineItemTest.php

```diff
@@ -71,14 +71,15 @@ public function testLineItemApiPermissions($version): void {
       'price_field_id' => $defaultPriceFieldID,
       'qty' => 1,
       'financial_type_id' => 'Donation',
-      'check_permissions' => TRUE,
+      'check_permissions' => ($version == 3),
     ];
     $line = $this->callAPISuccess('LineItem', 'Create', $lineParams);
     $lineParams['financial_type_id'] = 'Event Fee';
+    $lineParams['check_permissions'] = TRUE;
```

Comment:
This also suggests a slight change of behaviour  - ie the financialacls was potentially ignoring check_permissions & enforcing regarding before - I agree the change makes it more correct but once again it's probably worth ensuring @seamuslee001is aware of the subtle change in case it's worth discussing at JMA